src/img/glyph-close-16-inverted.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since we are only changing colors, we should add an #inverted ID to the existing SVGs instead of adding new files.